tycho-plugins/repository-utils/pom.xml

```diff
@@ -65,6 +65,12 @@
 			<artifactId>org.eclipse.jgit</artifactId>
 			<version>3.6.0.201411121045-m1</version>
 		</dependency>
+		<!-- Available under Apache License 2.0. http://www.lingala.net/zip4j/about.php  -->
```

Comment:
simpler mechanism for creating zips. See the places I reference net.lingala.zip4j below.